medusa/storage/__init__.py

```diff
@@ -44,6 +44,11 @@
 INDEX_BLOB_WITH_TIMESTAMP_PATTERN = re.compile('.*(started|finished)_(.*)_([0-9]+).timestamp$')
 
 
+def divide_chunks(values, step=64):
```

Comment:
Why set a default value for `step` here?
The method could be used for dealing with other things than splitting files to upload to GCP, where `64` would not make sense.
Since you already defined a constant for the GSUTIL chunk size and use it, I'd be keen to remove the default value to `step`.
Sounds good?

Reply:
Ok, I've fixed this.